src/components/package-list-item/package-list-item.css

```diff
@@ -2,13 +2,15 @@
   height: 100%;
 
   &.is-selected {
-    background-color: rgba(0,0,0,.5);
-    color: #FFFFFF;
+    background-color: rgba(0, 0, 0, 0.5);
+    color: #fff;
+
     &:hover {
-      background-color: rgba(0,0,0,.5);
+      background-color: rgba(0, 0, 0, 0.5);
     }
+
     & h5 {
-      color: #FFFFFF;
+      color: #fff;
```

Comment:
hm, won't this provide less accuracy than the full hex value?

Reply:
In this case, no. I think the stylelint rule only wants the three-character version for colors that have an accurate three-character translation. That makes for a few less characters of css.